website/docs/tutorials/07-testing/README.md

```diff
@@ -159,3 +159,53 @@ func test_something(a: felt) {
     return ();
 }
 ```
+
+### Custom python modules in hints
+
+You might want to use some non-standard python code in your hint segments to i.e. verify a signature using an external library.
+
+In order to do that, you should use `PYTHONPATH` env variable (https://docs.python.org/3/using/cmdline.html#envvar-PYTHONPATH), and protostar will consume it to extend `sys.path` accordingly.
+This approach will make it possible to include some packages from your virtual environment (by adding `site_packages` to your `PYTHONPATH`).
+
+Another approach is leveraging that project's `cairo-path` is added to `sys.path`, so you can put your python code alongside your cairo files.
```

Comment:
I think I would prefer using cairo path over python path, so I would move this paragraph up.

Reply:
Done